src/app/components/elements/SaveLogin.jsx

```diff
@@ -1,42 +0,0 @@
-import React, {Component, PropTypes} from 'react'
```

Comment:
where'd this file go?

Reply:
good q... removed by @valzav i think in the first batch of changes; it's not referenced anywhere in the app:

```
bc@zebrina:~/steemit/condenser$ grep -r SaveLogin src
bc@zebrina:~/steemit/condenser$ 
```

Reply:
@valzav what did this file do and why is it gone?

Reply:
SaveLogin was not in use